src/pykeen/evaluation/rank_based_evaluator.py

```diff
@@ -367,6 +376,9 @@ def process_head_scores_(
 
     def _get_ranks(self, side: ExtendedTarget, rank_type: ExtendedRankType) -> np.ndarray:
         if side == SIDE_BOTH:
+            # Even though it's been enabled to rank relations, this
+            # remains not having the relation label because it's not
+            # the standard evaluation technique
```

Comment:
I think this should go to the documentation of `SIDE_BOTH`, which means head + tail, instead of head + tail + relation. The latter does not make much sense, since the candidates are (a subset of) all entities for head/tail, but all relations for the relation prediction, and thus the ranks incomparable.